test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -159,6 +160,7 @@ var (
 		gvr("authorization.k8s.io", "v1beta1", "selfsubjectrulesreviews"):   `{"metadata": {"name": "", "namespace":"` + testNamespace + `"}, "spec": {"namespace":"` + testNamespace + `"}}`,
 
 		// Other Non persistent resources
+		gvr("", "v1", "serviceaccounts/token"): `{"metadata": {"name": "sa1name"}, "spec": {"audience": ["api"]}}`,
```

Comment:
this is probably better done as an addition to customTestFuncs (similar to something like testPodBindingEviction)... I don't think the subresource create implementation is likely to work well as a generic implementation

Reply:
done